app/src/components/main/modal/darkModalLayout/darkModalLayout.jsx

```diff
@@ -38,6 +39,11 @@ const messages = defineMessages({
   seeLess: { id: 'DarkModalLayout.seeLess', defaultMessage: 'See less' },
 });
 
+const BREAKPOINTS = {
+  HIDE_RIGHT_SECTION: 720,
+  HIDE_RIGHT_SECTION_BTN: 900,
```

Comment:
Why did you choose these specific values for breakpoints? Do our standard breakpoints suitable for this purpose?
$SCREEN_XXS_MAX: 320px;
$SCREEN_XS_MAX: 767px;
$SCREEN_SM_MAX: 991px;